spec/ad_parser.spec.js

```diff
@@ -187,12 +188,49 @@ describe('AdParser', function() {
         ad.adVerifications[2].trackingEvents.verificationNotExecuted
       ).toEqual([
         'http://example.com/verification-not-executed-EXE',
-        'http://sample.com/verification-not-executed-EXE'
+        'http://sample.com/verification-not-executed-EXE',
       ]);
     });
   });
 
-  describe('adVerifications from extensions', function() {
+  describe('parse adVerifications from extensions and parent node', function () {
+    let adVerificationsFromAd, ad;
+
+    beforeAll(() => {
+      adVerificationsFromAd = getNodesFromXml(
+        adVerificationFromExtensionsNodeAndFromNode
+      );
+      ad = parseAd(adVerificationsFromAd, null).ad;
+    });
+
+    it('should have 4 adVerifications', () => {
```

Comment:
The title is not correct, `adVerications` should contains 4 `adVerification` (singular).

Reply:
Thank you for your comments, I have fixed the issues :) Thanks for your approval